nengo_gui/components/sim_control.py

```diff
@@ -48,16 +48,17 @@ def finish(self):
         self.page.finish()
 
     def control(self, t):
+        self.actual_model_dt = t - self.time
         self.time = t
         self.sim_ticks += 1
 
-        now = time.time()
+        now = time.clock()
```

Comment:
Apparently, `time.clock()` [is deprecated in Python 3.3](http://stackoverflow.com/questions/85451/python-time-clock-vs-time-time-accuracy)